pyvista/core/_validation/_cast_array.py

```diff
@@ -99,7 +99,7 @@ def _cast_to_numpy(
             * is a subclass of ``np.ndarray`` and ``as_any`` is ``False``.
 
     must_be_real : bool, default: True
-        Raise a TypeError if the array does not have real numbers, i.e.
+        Raise a ``TypeError`` if the array does not have real numbers, i.e.
```

Comment:
Since the docstring modification has nothing to do with this PR, I have separated it to #6227 and #6228.  Maybe I am being too particular, but I want to keep the PR differential beautiful.

Reply:
I don't think this is a fruitful way of spending time, and it's a pretty bad experience for a contributor when PRs get micro-managed like this. Even if you feel so strongly about this we can discuss and you can let me handle my own work. Same issue why I was bothered by you pulling out a random comment into a new issue (without editing the issue to at least give it appropriate context).

Practicality beats purity and for sure we've historically mixed in unrelated but importat stuff in PRs. For instance, it would've been much simpler and more transparent to fix the recent broken main CI in a single PR rather than ~5 parallel ones, each with red builds of their own.

You do know I care about purity where it matters: I kept badgering others to keep auto-format PRs clean to preserve our git blame.

Reply:
I see. Perhaps I am being a bit dogmatic. Your pointed comments are very helpful. Please keep the advice coming.